mavros_extras/src/plugins/mount_control.cpp

```diff
@@ -209,13 +218,14 @@ class MountControlPlugin : public plugin::PluginBase {
 	 */
 	void handle_mount_orientation(const mavlink::mavlink_message_t *msg, mavlink::common::msg::MOUNT_ORIENTATION &mo)
 	{
-		auto q = ftf::quaternion_from_rpy(Eigen::Vector3d(mo.roll, mo.pitch, mo.yaw) * M_PI / 180.0);
+		const auto timestamp = ros::Time::now();
+		mo.pitch = -mo.pitch; // pitch angle is inverted to fix a gremsy implementation bug
```

Comment:
I am not sure if this is generally applicable and seems like a specific problem woth gremsy gimbals

Reply:
I made it parameter configurable, so it will workaround any existing and/or future gimbal quirks